packages/tools-iottwinmaker/src/commands/nuke.spec.ts

```diff
@@ -185,3 +195,53 @@ it('deletes all twinmaker resources when given a populated workspace', async ()
     sceneId: oneSceneListScenesResp.sceneSummaries![0].sceneId,
   });
 });
+
+it('deletes all twinmaker resources, s3 bucket, IAM role, workspace when given delete-workspace and delete-s3-bucket flags', async () => {
```

Comment:
Does command still delete IAM role? If not please rename test case

Reply:
Will update!